### PR TITLE
Add the empty "Replace" column in csv

### PR DESCRIPTION
**Pull request by v-xuto**
1.Create a new column named: "Replace".

@jongio for notification.

### CODE REVIEW

**Comment by weshaggard** [comment]
@jongio Should we go ahead prompt this data to it's own field in the CSV? Something like OtherPackage and we would know replaces/replaced by based on the New field.

**Comment by jongio** [comment]
Here's what I recommend:

1. Create a new column named: Replace
2. If more than one package, like with WindowsAzure.Storage, delimit them with a ","

**Comment by jongio** [comment]
@v-xuto / @tzhanl - Please do the following:

1. Create a new column named: "Replace"
2. Move the Notes column values to the new Replace column
3. If there are multiple values, then delimit them with a comma.
4. Remove the data from Notes (be careful not to remove Notes not related to package replacement, there are a few rows with other Notes).

Thank you.

**Comment by jongio** [comment]
@v-xuto - I think the quotes are required even if the fields are empty.  I wouldn't remove them just in case some system depends on them being there.

**Comment by v-xuto** [comment]
@Jongio - Thanks for your reminder, I will update.

**Comment by v-xuto** [comment]
@jongio - I have updated based on your comments.

File: _data/releases/latest/dotnet-packages.csv [hunk 1]
```diff
@@ -1,644 +1,644 @@
...
-"Microsoft.WindowsAzure.Management.StorSimple","","1.0.5-preview","WindowsAzure Management - StorSimple","StorSimple","NA","NA","NA","mgmt","false","","","","",""
-"Microsoft.WindowsAzure.Management.TrafficManager","","0.16.1-preview","WindowsAzure Management - Traffic Manager","Traffic Manager","NA","NA","NA","mgmt","false","","","","",""
-"Microsoft.WindowsAzure.Management.VirtualNetworks","","0.9.9-preview","WindowsAzure Management - Virtual Networks","Virtual Network","NA","NA","NA","mgmt","false","","","","",""
-"Microsoft.WindowsAzure.Management.WebSites","","5.0.0-prerelease","WindowsAzure Management - WebSites","App Service","NA","NA","NA","mgmt","false","","","","",""
+"Package","VersionGA","VersionPreview","DisplayName","ServiceName","RepoPath","MSDocs","GHDocs","Type","New","PlannedVersions","FirstGADate","Support","Hide","Replace","Notes"
+"Azure.AI.AnomalyDetector","","3.0.0-preview.2","Anomaly Detector","Anomaly Detector","anomalydetector","","","client","true","","","","","Replaces: Microsoft.Azure.CognitiveServices.AnomalyDetector",""
```
Comment (weshaggard) [inline _data/releases/latest/dotnet-packages.csv:2]: When you move to the Replace column you should remove the "Replaces: " or "Replaced by: " text.

**Comment by weshaggard** [comment]
Just to make diffing easier could you split this up into 2 PR's or at least 2 commits with the first one only adding the empty Replace column and then the second one moving the replace notes into the replace column without the "Replaces" text in the values.

**Comment by weshaggard** [comment]
BTW if it is helpful I've added a helper Powershell function to add a new column to all the csv files see https://github.com/Azure/azure-sdk/blob/master/eng/scripts/PackageList-Helpers.ps1#L95.

**Comment by v-xuto** [comment]
@weshaggard - According to your advice, I plan to post 2 PRs, the first one will only add an empty “Replace” column. Because it is easy to conflict with the master code. When this PR merged in, I will post a second PR about moving the replace notes into the replace column. Please Review this PR.

**Review by weshaggard — APPROVED** [review]

**Comment by v-xuto** [comment]
@weshaggard - The second PR https://github.com/Azure/azure-sdk/pull/2642 about moving the replace notes into the replace column. Please Review.

**Pull request merged**